base/install_area_detector.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+set -ex
+
+git clone --depth 1 --branch ${AREA_DETECTOR_VERSION} \
+    https://github.com/areaDetector/areaDetector
+
+cd areaDetector
+
+git submodule update --init --depth 1 -j ${JOBS} \
+    ADSupport \
+    ADCore
+
+cd configure
+
+module_releases="
+AREA_DETECTOR=${EPICS_MODULES_PATH}/areaDetector
+ADSUPPORT=${EPICS_MODULES_PATH}/areaDetector/ADSupport
+ADCORE=${EPICS_MODULES_PATH}/areaDetector/ADCore
+"
+
+echo "$module_releases" >> ${EPICS_MODULES_PATH}/../RELEASE
+
+echo "
+EPICS_BASE=${EPICS_BASE_PATH}
+
+$module_releases
+
+ASYN=${EPICS_MODULES_PATH}/asyn
+" > RELEASE.local
+
+ln -s RELEASE.local RELEASE_PRODS.local
+ln -s RELEASE.local RELEASE_LIBS.local
+
+echo "
+BUILD_IOCS=NO
+
+WITH_BOOST=NO
+WITH_PVA=NO
+WITH_QSRV=NO
```

Comment:
Why not build the QSRV plugin, if we are already here?

Reply:
Only because I don't know any easy way to test if it is working properly yet. PV Access is still completely new for me. Should we install anything else to make it work or should it be just enable it in this configuration?

Reply:
Building with qsrv only allows us to use the pvaccess plugin, which is opt-in. I'd rather have it built than not.